### PR TITLE
force mpl2 and remove seaborn

### PR DESCRIPTION
**Pull request by ocefpaf**
Not ready for merging yet b/c I could not run the notebook until the end. See https://github.com/geohackweek/vector/issues/18#issuecomment-338751460

### CODE REVIEW

File: vector/environment.yml [hunk 1]
```diff
@@ -10,5 +10,5 @@ dependencies:
 - rasterstats
 - requests
 - folium
-- seaborn
-- jupyter
\ No newline at end of file
+- matplotlib=2
```
Comment (ocefpaf) [inline vector/environment.yml]: I fixed all the package upstream and this is not really needed, I only added it here to ensure mpl 2 will be installed.

**Comment by emiliom** [comment]
Thanks, @ocefpaf!

Actually, my goal is *not* to replace the current notebooks with ones that don't use PostGIS. So, I think I'll go over your notebooks (from your fork) and compare manually to the original, PostGIS-based ones to examine the differences in the plot style results and style parameters. Then port your changes manually (it'd be much nicer if there were good notebook diffing tools! Or, I need to look into [nbdiff](http://nbdiff.org), eventually)

I'm in a conference call right now and have another one coming up. So I won't do this until a few hours from now. But I looked over quickly at the intro notebook and it looked good. I haven't looked at the second notebook yet.

**Comment by emiliom** [comment]
Since I didn't say this explicitly: I won't be merging this PR. But let's leave it here for now, to help the comparisons.

**Comment by ocefpaf** [comment]
>  Or, I need to look into nbdiff, eventually)

[`conda install nbdime`](http://nbdime.readthedocs.io/en/latest/) and be happy :smile: 

> But I looked over quickly at the intro notebook and it looked good. I haven't looked at the second notebook yet.

They use latest `matplotlib` and `geopandas` defaults, which are, IMO, better than `seaborn`.
I'll add a new commit restoring the notebooks to the remote data, without the `seaborn` stuff, so you can have a clean merge. However, someone with access to the data should run than and commit to get the new images.

**Comment by ocefpaf** [comment]
I reverted the commit that added the last commit that uses the local data but left the changes that removes seaborn. You can still see the new images in [geopandas_advanced.ipynb ](http://nbviewer.jupyter.org/github/ocefpaf/tutorial_contents/blob/aa505496c967b21b7365909d3d8747af97fa2107/vector/notebooks/geopandas_advanced.ipynb) and [geopandas_intro.ipynb](http://nbviewer.jupyter.org/github/ocefpaf/tutorial_contents/blob/aa505496c967b21b7365909d3d8747af97fa2107/vector/notebooks/geopandas_intro.ipynb).

**Comment by emiliom** [comment]
Thanks!! I'll do the testing myself, before merging. Tomorrow.

> conda install nbdime and be happy

Yeah, I read about that a couple of months ago, but haven't had the opportunity to test it. Looks promising. Have you used that suite of tools?

**Comment by ocefpaf** [comment]
> Have you used that suite of tools?

I did the `git diff` configuration and use it everyday.

**Comment by lsetiawan** [comment]
@emiliom Trying to clean things to prepare for geohackweek, should I close of merge this PR? Thanks.

**Comment by emiliom** [comment]
Would you mind if we discuss it on Friday (or tomorrow, if you're working on it then), before making a decision?

**Comment by emiliom** [comment]
@lsetiawan I'll try to review the status of this later this afternoon, and get back to you.

**Comment by emiliom** [comment]
@lsetiawan I've gone over this, and will close the PR right away.

**Pull request closed without merging**